src/views/CampusSafety/components/Header/index.tsx

```diff
@@ -1,66 +1,90 @@
       {children}
     </Typography>
   );
 };
 
-// Page header component, has a option for the safety page, and an option that holds any
-// child element for other pages
-const Header = ({ safetyPage }: { safetyPage?: boolean }, children: JSX.Element | null) => {
-  const location = useLocation().pathname;
-  // console.log(typeof location);
-  return safetyPage || safetyPage === undefined ? (
-    <>
-      <Grid container alignItems="center" columnSpacing={4} className={styles.headerMain}>
-        <Grid item container xs={9} alignItems="center" columnSpacing={2}>
-          <Grid item xs={8}>
-            <Typography className={styles.title}>
-              <Box component="span" sx={{ color: 'secondary.main' }}>
-                Gordon
-              </Box>{' '}
-              Campus Safety Resources
-            </Typography>
-            <Typography className={styles.subtitle}>
-              <i>"Helping students help themselves"</i>
-            </Typography>
+// Define the props for Header
+type HeaderProps = {
+  safetyPage?: boolean;
+  children?: React.ReactNode;
+};
+
+const Header: React.FC<HeaderProps> = ({ safetyPage = true, children }) => {
+  const location = useLocation();
+  const pathnames = location.pathname.split('/').filter((x) => x);
+
+  if (safetyPage) {
+    return (
+      <>
+        <Grid container alignItems="center" columnSpacing={4} className={styles.headerMain}>
+          <Grid item container xs={9} alignItems="center" columnSpacing={2}>
+            <Grid item xs={8}>
+              <Typography className={styles.title}>
+                <Box component="span" sx={{ color: 'secondary.main' }}>
+                  Gordon
+                </Box>{' '}
+                Campus Safety Resources
+              </Typography>
+              <Typography className={styles.subtitle}>
+                <i>"Helping students help themselves"</i>
+              </Typography>
+            </Grid>
           </Grid>
         </Grid>
-      </Grid>
-      <AppBar className={styles.stickyNav}>
-        <Breadcrumbs aria-label="breadcrumb">
-          {/* Home breadcrumb */}
-          <CampusSafetyBreadcrumb link={location !== '/campussafety' ? `/campussafety` : null}>
-            <Grid container alignItems="center">
-              <HomeIcon sx={{ mr: 0.5 }} fontSize="inherit" />
-              {'Campus Safety Home'}
-            </Grid>
-          </CampusSafetyBreadcrumb>
-        </Breadcrumbs>
-      </AppBar>
-    </>
-  ) : (
-    children && <Grid className={styles.mainHeader}>{children}</Grid>
-  );
+        <AppBar className={styles.stickyNav}>
+          <Breadcrumbs
+            aria-label="breadcrumb"
+            separator={<NavigateNextIcon fontSize="small" />}
+            className={styles.breadcrumbContainer}
+          >
+            {/* Home breadcrumb */}
+            <CampusSafetyBreadcrumb link={pathnames.length > 0 ? '/campussafety' : null}>
```

Comment:
This line should probably check if the pathname length is > 1, not > 0.  With this, the home page breadcrumb is a link even when you are on the home page, which there is purposeless since you're already on the page it links to.

Reply:
Yup i can change that, that was an oversight